src/app/components/stepOne/FindPatient.tsx

```diff
@@ -0,0 +1,65 @@
+import style from './FindPatient.module.css'
+import { useState } from 'react'
+import { fetchPatientData } from '../functions/fetchPatientData';
+import { useStepsContext, STEPS_NAMES } from '@/app/context/stepsContext';
+import { usePatientContext } from '@/app/context/patientContext';
+import { Loading } from '../Loading';
+
+export function FindPatient() {
+    const [isLoading, setIsLoading] = useState<boolean>(false)
+    const value = useStepsContext()
```

Comment:
```suggestion
    const stepValue = useStepsContext()
```
generally worth having descriptive variable names